lib/galaxy/config.py

```diff
@@ -305,7 +305,7 @@ def __init__( self, **kwargs ):
             self.use_tool_dependencies = True
         else:
             self.tool_dependency_dir = None
-            self.use_tool_dependencies = False
+            self.use_tool_dependencies = os.path.exists(self.dependency_resolvers_config_file)
```

Comment:
Previously, to use Tool Shed or manual tool dependencies, it was enough to change the `tool_dependency_dir` directive in `config/galaxy.ini`, now also this sample file needs to be copied.